.cbmc-batch/source/make_common_data_structures.c

```diff
@@ -146,3 +146,9 @@ enum aws_cryptosdk_sha_version aws_cryptosdk_which_sha(enum aws_cryptosdk_alg_id
         default: return AWS_CRYPTOSDK_NOSHA;
     }
 }
+
+void ensure_cryptosdk_keyring_has_allocated_members(
+    struct aws_cryptosdk_keyring *keyring, const struct aws_cryptosdk_keyring_vt *vtable) {
+    keyring->refcount.value = can_fail_malloc(sizeof(size_t));
+    keyring->vtable         = nondet_bool() ? NULL : &vtable;
```

Comment:
This worries me.  Unless I'm reading it wrong, you're taking the address of a stack variable, that will drop out of scope after this function? Did you mean just `NULL : vtable`?